packages/dashboard/src/app/api/useStoryApi.js

```diff
@@ -127,33 +121,17 @@ const useStoryApi = (dataAdapter, { storyApi, encodeMarkup }) => {
         }
         isInitialFetch.current = false;
 
-        // Hardening in case data returned by the server is malformed.
-        // For example, story_data could be missing/empty.
-        const cleanStories = response.body.filter((story) =>
-          Array.isArray(story?.story_data?.pages)
-        );
+        const stories = response.body;
```

Comment:
This is no longer an edge case?  https://github.com/google/web-stories-wp/pull/6095

Reply:
The field is not longer in the api to make responses faster. We also do not use that story data for anything anymore. So safe to remove.